tests/Doctrine/Tests/DBAL/Functional/BlobTest.php

```diff
@@ -49,6 +51,27 @@ public function testInsert()
         self::assertEquals(1, $ret);
     }
 
+    public function testInsertProcessesStream()
+    {
+        if (in_array($this->_conn->getDatabasePlatform()->getName(), ['oracle', 'db2'], true)) {
+            // https://github.com/doctrine/dbal/issues/3288 for DB2
+            $this->markTestSkipped('Platform does not support stream resources as parameters');
```

Comment:
Nitpicking: both platforms do support streams, their support is just not implemented in DBAL. Therefore, the test should be marked as incomplete (therefore, requiring a fix), not skipped (would be correct if the platforms really didn't support the feature).

Reply:
fixed.